packages/backend/.env

```diff
@@ -0,0 +1,6 @@
+# used to point external links at our frontend i.e. in email body.
+VITE_FRONTEND_URL="http://localhost:3000"
+# used by emailNotificationService
+VITE_EMAIL_NOTIFIER_HOST="smtp.gmail.com"
+VITE_EMAIL_NOTIFIER_EMAIL="fake-email"
+VITE_EMAIL_NOTIFIER_PASSWORD="fake-password"
```

Comment:
Hmm.  I don't want to check in secrets in the .env

But we will have to figure out a way to pass env.  Maybe we just don't check the .env.development / .env.production into source code, and we manually fill these notifier user / pass here.  

We will generate these values with google mail service because it's the easiest to make a new gmail & get things setup.